flasc/energy_ratio/energy_ratio.py

```diff
@@ -381,6 +405,29 @@ def compute_energy_ratio(er_in: EnergyRatioInput,
     # Convert the numbered arrays to appropriate column names
     test_cols = [f'pow_{i:03d}' for i in test_turbines]
 
+    # If df_freq is provided, confirm is consistent with ws/wd min max and
+    # prepare a polars table of weights
+    if df_freq is not None:
+
+        # Maybe not test, not sure yet
```

Comment:
I think it would be good to check the match between the ws/wd min, max, step bins and those in df_freq here. Maybe this could raise an error if none of the bin centers given by ws/wd min, max, step are included in df_freq, and print a warning message if some but not all of the ws/wd min, max, step bins are included in df_freq.

Reply:
This is a good idea, I think why I had paused this earlier and left as a todo is what is the best way to check?  Should I just make sure that at least 1 wd bin and 1 ws bin match?  Or should I confirm that we have a match on at least 1 (wd,ws) combination?  Maybe that is the best approach, convert the unique ws/wd combinations into a list of tuples and do a set difference?  Then check the length, if 0 raise error, < the length of the ws/wd min, max, step raise warning?

Reply:
I hadn't thought about it, but now that you mention the options, I think it makes sense to check for matches on the (wd,ws) combinations, and the error vs. warning logic you described sounds like a good approach to me.

Reply:
Ok, believe I've now caught this case more generally and added a test to confirm